app/src/components/common/Table.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="table-container">
+    <table class="user-table">
+      <thead v-if="columns.length > 0">
+        <tr>
+          <th
+            v-for="column in columns"
+            :key="column.key"
+          >
+            {{ column.label }}
```

Comment:
In the [TableColumn](https://github.com/SolomiiaLomnytska22/solomiia-training/pull/24/files#diff-a7cb6a9784f8cbe81a0612f4c0b0dfd638f12606d2379927c091662cad31c7b3R39) the `label` key is optional. Should we consider it in the template? Otherwise, let's make it required.